src/gamecontroller.cs

```diff
@@ -186,6 +203,9 @@ partial interface GCExtendedGamepad {
 		[Export ("valueChangedHandler", ArgumentSemantic.Copy)]
 		GCExtendedGamepadValueChangedHandler ValueChangedHandler { get; set; }
 
+		[Deprecated (PlatformName.MacOSX, 10, 15, message: "GCExtendedGamepadSnapshot has been deprecated, use [GCController capture] instead")]
+		[Deprecated (PlatformName.iOS, 13, 0, message: "GCExtendedGamepadSnapshot has been deprecated, use [GCController capture] instead")]
+		[Deprecated (PlatformName.TvOS, 13, 0, message: "GCExtendedGamepadSnapshot has been deprecated, use [GCController capture] instead")]
```

Comment:
Hello, is this part of the message ```GCExtendedGamepadSnapshot has been deprecated``` worth having ? I thought we just needed ```use xxx instead```

Reply:
yes, we can't use the text from header files
As much as possible (for direct replacements) we try to stick to this format: `"Use 'XXX' instead."`, where `XXX` is the *managed* name of the API providing the replacement

Reply:
Good catch! I will fix these, thanks!